### PR TITLE
MP-75 Copy metrics.xml & hazelcast-config.xml config files to MICRO-INF domain folder

### PR DESCRIPTION
**Pull request by dhutchison**
See https://github.com/payara/ecosystem-maven/issues/103

### CODE REVIEW

**Comment by jGauravGupta** [comment]
Hi @dhutchison,

Instead of introducing another configuration element, May you add `commandLineOptions` to `BundleMojo`, so `--rootDir` option can be used in `payara-micro:bundle` goal.
        Also `CustomFileCopyProcessor` can be improved further by explicitly copying the commonly used config files like metrics.xml, etc.

**Comment by dhutchison** [comment]
Looking at what StartMojo does with commandLineOptions, it is being used as part of building up an array of commands which can ultimately be passed to Runtime.exec() and invoked on the payara micro jar. 

This is a different model to how the BundleMojo works, which does all the work as part of the Mojo without starting Payara micro. So to implement commandLineOptions here, the options supplied would require to be parsed, and each possible command line option implemented in a BaseProcessor instance as part of the Mojo. Is that what you are suggesting as an alternative to a config element?

Metrics.xml can be added to the pre-existing list - is there a reference somewhere of other commonly used config files payara micro looks for that this should be including? Adding metrics.xml to the default list covers the use case I was needing. I hadn't seen any documentation covering the 4 files it was already copying, only found out about them through looking through the source code.

File: payara-micro-maven-plugin/README.md [hunk 1]
```diff
@@ -44,6 +44,10 @@ This goal bundles the attached project's artifact into uber jar with specified c
                     <version>6.0</version>
                 </artifactItem>
             </customJars>
+            <customConfigs>
```
Comment (jGauravGupta) [inline payara-micro-maven-plugin/README.md]: `<customConfigs>` element should be removed from example.

**Comment by jGauravGupta** [comment]
Hi @dhutchison,

Yes, BundleMojo base processing is different than StartMojo, so to address the --rootDir support in payara-micro:bundle goal, I have created internal ticket MP-72.

List of the config file is here: https://github.com/payara/Payara/blob/master/nucleus/cluster/admin/src/main/resources/META-INF/config-files#L45 and adding `metrics.xml` & `hazelcast-config.xml` to default copied configs, absolutely makes more sense, thanks for that.
Also created internal ticket MP-73 & MP-74 for documentation of default config and boot command files.

File: payara-micro-maven-plugin/README.md (hunk 1 above)
Comment (dhutchison) [reply payara-micro-maven-plugin/README.md]: Now removed

**Pull request merged**